crates/router/src/connector/authorizedotnet/transformers.rs

```diff
@@ -341,12 +368,27 @@ impl TryFrom<&AuthorizedotnetRouterData<&types::PaymentsAuthorizeRouterData>>
             }),
             None => None,
         };
-
+        let bill_to = match item.router_data.get_billing() {
+            Ok(billing) => billing.address.as_ref().map(|address| BillTo {
+                first_name: address.first_name.clone(),
+                last_name: address.last_name.clone(),
+                address: address.line1.clone(),
+                city: address.city.clone(),
+                state: address.state.clone(),
+                zip: address.zip.clone(),
+                country: address.country,
+            }),
+            Err(_) => None,
+        };
         let transaction_request = TransactionRequest {
             transaction_type: TransactionType::from(item.router_data.request.capture_method),
             amount: item.amount,
-            payment: payment_details,
             currency_code: item.router_data.request.currency.to_string(),
+            payment: payment_details,
+            order: Order {
+                description: item.router_data.connector_request_reference_id.clone(),
```

Comment:
will this reflect for void/capture payments

Reply:
Yes

Reply:
<img width="588" alt="Screenshot 2024-03-06 at 5 29 03 PM" src="https://github.com/juspay/hyperswitch/assets/41580413/1cf1e5a2-6010-4d94-a4da-c68b5368ae64">